buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -69,7 +69,7 @@ private fun Project.registerLiquibaseTask(projectName: String, relativeChangeLog
             "--rm",
             "--env", "INSTALL_MYSQL=true",
             "--network", "build_default",
-            "liquibase/liquibase:4.15",
+            "liquibase/liquibase:4.20",
```

Comment:
what's about `liquibase/liquibase` in the rest deployments?

Reply:
There is no problem with the other deployment. Will change if really required

Reply:
Done